src/rubintv/static/js/refresh.js

```diff
@@ -1,9 +1,9 @@
-$(document).ready(function(){
-      refreshDiv();
-    });
+$(document).ready(function () {
+  refreshDiv();
+});
 
-    function refreshDiv(){
-        $('#refresher').load(window.location.href + " #refresher" , function(){
-           setTimeout(refreshDiv, 500);
-        });
-    }
\ No newline at end of file
+function refreshDiv() {
+  $('#refresher').load(window.location.href + ' #refresher', function () {
+    setTimeout(refreshDiv, 10000);
```

Comment:
One thing this sets is how fast the 'monitor' channels refresh.  This is important for people waiting for the most recent image to show up.  I think people may notice a 10 second delay.  Do you think it's possible to push this down to `5000` ms, or will that result in a query backlog, do you think?

Reply:
5 seconds could work. Anything faster and I'd recommend re-writing Rubin TV around a more sound architecture.